api.planx.uk/modules/send/bops/bops.ts

```diff
@@ -41,11 +41,22 @@ const sendToBOPS: SendIntegrationController = async (_req, res, next) => {
       encryptionKey: process.env.ENCRYPTION_KEY!,
       env,
     });
+
     const exportData = await $api.export.digitalPlanningDataPayload(sessionId);
+    const applicationType = exportData?.data?.application?.type?.value;
+
+    // Enforcement submissions use a different BOPS endpoint than planning applications (in future all planned to use `/submissions`)
+    let bopsSubmissionURLByAppType = bopsSubmissionURL;
+    if (applicationType === "breach") {
+      bopsSubmissionURLByAppType = bopsSubmissionURL?.replace(
+        "/planning_applications",
+        "/submissions",
+      );
+    }
```

Comment:
See context for this here https://opendigitalplanning.slack.com/archives/C0182MVK4AW/p1755160687052099